src/dnssec/sign/keys/keyset.rs

```diff
@@ -362,8 +417,12 @@ impl KeySet {
 
             // Set old for any key we find.
             keystate.old = true;
+
+            // Add algorithm
```

Comment:
Nit: "add" sounds like it is making a change to something.
```suggestion
            // Remember algorithm
```